auslib/blobs/apprelease.py

```diff
@@ -705,6 +705,25 @@ def __init__(self, **kwargs):
             self['schema_version'] = 5
 
 
+class ReleaseBlobV6(ReleaseBlobBase, NewStyleVersionsMixin, MultipleUpdatesXMLMixin, UnifiedFileUrlsMixin):
+    """
+    """
```

Comment:
Please add a comment with what's changed between V5 and V6. It makes things much easier for humans!